datasetinsights/estimators/factory.py

```diff
@@ -0,0 +1,85 @@
+from tensorboardX import SummaryWriter
```

Comment:
Do we have unit test case of this?

Reply:
Currently we do not have any unit tests for the Factory i will add some